EncoreTickets.ConsoleTester/Program.cs

```diff
@@ -1,44 +1,67 @@
-﻿using EncoreTickets.SDK;
-using EncoreTickets.SDK.Inventory;
+﻿using EncoreTickets.SDK.Inventory;
 using System.Collections.Generic;
 using System.Linq;
 using System;
 using EncoreTickets.SDK.Content;
-using Product = EncoreTickets.SDK.Inventory.Product;
-using Product2 = EncoreTickets.SDK.Content.Product;
+using Product = EncoreTickets.SDK.Inventory.Models.Product;
+using Product2 = EncoreTickets.SDK.Content.Models.Product;
 using EncoreTickets.SDK.Venue;
+using EncoreTickets.SDK.Api.Context;
+using EncoreTickets.SDK.Content.Models;
+using EncoreTickets.SDK.Inventory.Models;
 
 namespace SDKConsoleTester
 {
     class Program
     {
         static void Main(string[] args)
         {
-            ApiContext context = new ApiContext(Environments.Sandbox);
-            List<string> productIds = new List<string>();
-            context.affiliate = "encoretickets";
-            // uscontext.useBroadway = true;
+            Console.Write("Enter username: ");
+            var userName = Console.ReadLine();
+            Console.Write("Enter Password: ");
+            var password = Console.ReadLine();
+            var context = new ApiContext(Environments.Sandbox, userName, password) {Affiliate = "encoretickets"};
+            // context.UseBroadway = true;
```

Comment:
I think it makes sense to add additional comment why we need this unavailable code and when should we uncomment it

Reply:
actually, we can remove the UseBroadway altogether.  This was to support a workaround in the API that required a header to inform the routing logic in the service, but this is now dynamic so we don't need support for it in the SDK.